RepoZ.App.Win/i18n/de-DE.xaml

```diff
@@ -37,7 +37,8 @@ Alternativ, kann der Computer auch über das Menü oben rechts nach Repositories
 	<system:String x:Key="Open in Git Bash">In Git Bash öffnen</system:String>
 	<system:String x:Key="Open in Finder">In Finder öffnen</system:String>
 	<system:String x:Key="Open in Terminal">Im Terminal öffnen</system:String>
-	<system:String x:Key="Fetch">Git Fetch</system:String>
+    <system:String x:Key="Open {0}">Öffnen {0}</system:String>
```

Comment:
In German, we'd say "RepoZ.sln öffnen", so it would be great if you could change this to "{0} öffnen" (with a small "ö")

Reply:
Ah, my German classes are slowly coming back to me now. Veeery slowly... :)